reactor-core/src/test/java/reactor/core/publisher/LambdaMonoSubscriberTest.java

```diff
@@ -165,18 +165,38 @@ public void onNextConsumerExceptionBubblesUpDoesntTriggerCancellation() {
 		TestSubscription testSubscription = new TestSubscription();
 		tested.onSubscribe(testSubscription);
 
-		//as Mono is single-value, it cancels early on onNext. this leads to an exception
-		//during onNext to be bubbled up as a BubbledException, not propagated through onNext
+		//the error is expected to be propagated through doError
+		tested.onNext("foo");
+
+		assertThat(errorHolder.get()).as("onError").isInstanceOf(IllegalArgumentException.class);
+		assertThat(testSubscription.isCancelled).as("subscription isCancelled").isTrue();
+	}
+
+
+	@Test
+	public void onNextConsumerBubblesUpErrorCallbackNotImplemented() {
+		AtomicReference<Throwable> errorHolder = new AtomicReference<>(null);
+
+		LambdaMonoSubscriber<String> tested = new LambdaMonoSubscriber<>(
+				value -> { throw new IllegalArgumentException(); },
+				null,
+				() -> {},
+				null);
+
+		TestSubscription testSubscription = new TestSubscription();
+		tested.onSubscribe(testSubscription);
+
+		//the error is expected to be thrown as there is no error handler
 		try {
 			tested.onNext("foo");
-			fail("Expected a bubbling Exception");
-		} catch (RuntimeException e) {
-			assertThat(e).matches(Exceptions::isBubbling, "Expected a bubbling Exception")
-			             .hasCauseInstanceOf(IllegalArgumentException.class);
+			fail("Expected IllegalArgumentException to be thrown");
+		}
+		catch (UnsupportedOperationException e) {
```

Comment:
mmh `UnsupportedOperatorException` ?? 🤔

Reply:
also you could use assertj `assertThatIllegalArgumentException().isThrownBy(() -> tested.onNext("foo"))`